var/spack/repos/builtin/packages/ufs-weather-model/package.py

```diff
@@ -17,68 +17,160 @@ class UfsWeatherModel(CMakePackage):
+    variant(
+        "jedi_driver",
+        default=False,
+        description="Enable JEDI as top level driver",
+        when="@develop",
+    )
+    variant(
+        "cmeps_aoflux",
+        default=False,
+        description="Enable atmosphere-ocean flux calculation in mediator",
+        when="@develop",
+    )
+    variant(
+        "ccpp",
+        default=True,
+        description="Enable the Common Community Physics Package (CCPP)",
+        when="@:2.0.0",
     )
     variant(
         "ccpp_suites",
         default="FV3_GFS_v15p2,FV3_RRFS_v1alpha",
         description="CCPP suites to compile",
         values=("FV3_GFS_v15p2", "FV3_RRFS_v1alpha", "FV3_GFS_v15p2,FV3_RRFS_v1alpha"),
         multi=True,
+        when="@:2.0.0",
+    )
+    dev_ccpp_default = [
+        "FV3_GFS_v16",
+        "FV3_GFS_v15_thompson_mynn",
+        "FV3_GFS_v17_p8",
+        "FV3_GFS_v17_p8_rrtmgp",
+        "FV3_GFS_v15_thompson_mynn_lam3km",
+    ]
+    variant(
+        "ccpp_suites",
+        default=",".join(dev_ccpp_default),
+        description="CCPP suites to compile",
+        multi=True,
+        when="@develop",
     )
-    variant("inline_post", default=False, description="Compile post processing inline")
-    variant("multi_gases", default=False, description="Enable multi gases in physics routines")
-    variant("openmp", default=True, description="Enable OpenMP")
-    variant("parallel_netcdf", default=True, description="Enable parallel I/O in netCDF")
     variant(
         "quad_precision",
         default=False,
         description="Enable quad precision for certain grid metric terms in dycore",
+        when="@:2.0.0",
     )
+    variant("mom6solo", default=False, description="Build MOM6 solo executable", when="@develop")
+
     variant(
-        "simdmultiarch", default=False, description="Enable multi-target SIMD instruction sets"
+        "cmake_platform",
+        default="auto",
+        description="Override CMAKE_Platform ('linux.intel', 'hera.gnu', 'acorn', etc.)",
     )
 
+    variant("app", default="ATM", description="UFS application", when="@develop")
+
     depends_on("bacio")
-    depends_on("esmf@:8.0.0")
-    depends_on("mpi")
-    depends_on("nemsio")
+    depends_on("mpi", when="+mpi")
     depends_on("netcdf-c")
     depends_on("netcdf-fortran")
     depends_on("sp")
     depends_on("w3emc")
-    depends_on("w3nco")
+
+    depends_on("esmf@:8.0.0", when="@:2.0.0")
+    depends_on("nemsio", when="@:2.0.0")
+    depends_on("w3nco", when="@:2.0.0")
+
+    depends_on("bacio@2.4.0:", when="@develop")
+    depends_on("crtm", when="@develop")
+    depends_on("esmf@8.3.0:", when="@develop")
+    depends_on("fms@2022.04: constants=GFS", when="@develop")
+    depends_on("g2", when="@develop")
+    depends_on("g2tmpl", when="@develop")
+    depends_on("hdf5+hl+mpi", when="@develop")
+    depends_on("ip", when="@develop")
+    depends_on("netcdf-c~parallel-netcdf+mpi", when="@develop")
```

Comment:
Are these all truly `@develop` or is this in lieu of something like `@3:` where we just don't know what the next release number is yet?

Reply:
More or less- My expectation is that the typical use case for this package will be development off of the develop branch, which is why I haven't put in other versions.